include/swift/AST/DiagnosticsSema.def

```diff
@@ -81,7 +81,7 @@ ERROR(cannot_infer_keypath_root_anykeypath_context,none,
      "'AnyKeyPath' does not provide enough context for root type to be inferred; "
      "consider explicitly specifying a root type", ())
 
-ERROR(could_not_find_type_member,none,
```

Comment:
@JinYuchyi I think we should leave this as an error and change the other one to a note. 

Ideally, if we can suppress this completely then we can allow the earlier message to be an error. Until this error message show up, it should show up as an error.